x-pack/plugins/cloud_security_posture/server/create_transforms/create_transforms.ts

```diff
@@ -36,13 +38,14 @@ export const createTransformIfNotExists = async (
   try {
     await esClient.transform.getTransform({
       transform_id: transform.transform_id,
-      allow_no_match: true,
     });
+    return true;
```

Comment:
nit: it returns true although it was not created. consider documentation/ better naming for the function